src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/migration/NoticeTypeSeedJob.kt

```diff
@@ -0,0 +1,71 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.migration
+
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Pageable
+import org.springframework.data.domain.Slice
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1ApplicationNoticeType
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApprovedPremisesApplicationEntity
+import java.util.UUID
+
+@Repository
+interface NoticeTypeSeedApplicationRepository : JpaRepository<ApplicationEntity, UUID> {
+  @Modifying
+  @Query(
+    "UPDATE ApprovedPremisesApplicationEntity ap set " +
+      "ap.noticeType = 'emergency' " +
+      "where ap.isEmergencyApplication = true",
+  )
+  fun updateEmergencyApplicationStatus()
+
+  @Modifying
+  @Query("UPDATE ApprovedPremisesApplicationEntity ap set ap.noticeType = :status where ap.id in :ids")
+  fun updateApplicationStatus(ids: List<UUID>, status: Cas1ApplicationNoticeType)
```

Comment:
can we please rename this to updateApplicationNoticeType for clarity (and also change arg to noticeType)